source/_patterns/02-molecules/vue-widget/demo/vue-html-example.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <form 
+    id="vue-html-example-2" 
+    class="p-5"
+  >
+    <div class="input-group">
+      <div class="input-group-prepend">
+        <span class="input-group-text">You typed: <strong>{{ name }}</strong></span>
+      </div>
+      <input
```

Comment:
Input should have a corresponding label for accessibility. Since we are bolstering our ESLint config I would suggest potentially including [ESLint Vue a11y plugin](https://www.npmjs.com/package/eslint-plugin-vue-a11y) as well. I have had very positive experience with the corresponding React plugin.

Reply:
Sounds good. I appreciate having the label in as an accessibility best practice even for a sample page.